### PR TITLE
timer/layout integration

### PR DESCRIPTION
**Pull request by Margiorie**
#### What does this PR do?
Integrates timer with existing layout
...

screen shot 2018-10-11 at 15 29 28

### CODE REVIEW

File: app/views/layouts/_nav_layout.html.erb [hunk 1]
```diff
@@ -21,6 +21,15 @@
         </ul>
       </div>
       <div class="vertical-devider"></div>
+      <div class="timer-container">
+        <%= render partial: "timer/chronometer" %>
+      </div>
+      <div class="top-nav-button" id="projects-button">
+        <i class="fas fa-clock"></i></i>
```
Comment (lucaleivaloop) [inline app/views/layouts/_nav_layout.html.erb]: Double closing of the `i` tag

File: app/views/layouts/_nav_layout.html.erb [hunk 2]
```diff
@@ -21,6 +21,15 @@
...
+      </div>
+      <div class="top-nav-button" id="projects-button">
+        <i class="fas fa-hand-peace "></i></i>
```
Comment (lucaleivaloop) [inline app/views/layouts/_nav_layout.html.erb]: Double closing of the `i` tag

File: app/assets/stylesheets/side-nav.css [hunk 3]
```diff
@@ -103,19 +103,23 @@
   color: #4b4b4b;
 }
 
-.layout-spacer{
+.layout-spacer {
   position: fixed;
-  right: 5%;
-  display:flex;
+  right: 30px;
```
Comment (lucaleivaloop) [inline app/assets/stylesheets/side-nav.css]: Was tested that a fixed measure as `30px` is nice in all viewport sizes?

File: app/assets/stylesheets/timer.css [hunk 4]
```diff
@@ -1,13 +1,17 @@
-// Place all the styles related to the Timer controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
-
-.timer-container{
-  display: flex;
+.timer-container {
+  /* display: flex;
```
Comment (lucaleivaloop) [inline app/assets/stylesheets/timer.css]: Should avoid commented code

File: app/assets/stylesheets/side-nav.css (hunk 3 above)
Comment (Margiorie) [reply app/assets/stylesheets/side-nav.css]: screen shot 2018-10-11 at 15 58 34
screen shot 2018-10-11 at 15 58 44
screen shot 2018-10-11 at 15 58 57
screen shot 2018-10-11 at 15 59 13
yep

File: app/assets/stylesheets/timer.css [hunk 5]
```diff
@@ -54,11 +58,11 @@
 }
 
 .timer {
-  flex-basis: 20%;
-  height: 49px;
+  /* flex-basis: 20%;
```
Comment (lucaleivaloop) [inline app/assets/stylesheets/timer.css]: Should avoid commented code

Comment (Margiorie) [reply app/assets/stylesheets/timer.css]: ok!

File: app/assets/stylesheets/side-nav.css [hunk 6]
```diff
@@ -103,19 +97,16 @@
...
+  display: flex;
   justify-content: center;
+  align-items: center;
+  display: flex;
```
Comment (mconiglio) [inline app/assets/stylesheets/side-nav.css]: `display: flex` appear two times

File: app/assets/stylesheets/side-nav.css [hunk 7]
```diff
@@ -103,19 +97,16 @@
...
 }
 
-.layout-spacer{
+.layout-spacer {
   position: fixed;
-  right: 5%;
-  display:flex;
+  right: 30px;
+  display: flex;
   justify-content: center;
```
Comment (mconiglio) [inline app/assets/stylesheets/side-nav.css]: `justify-content` prop is overriden 3 lines below

**Pull request merged**